Assets/MixedRealityToolkit/_Core/Utilities/Lines/Renderers/MeshLineRenderer.cs

```diff
@@ -0,0 +1,233 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities.Lines.Renderers
+{
+    public class MeshLineRenderer : BaseMixedRealityLineRenderer
+    {
+        private const string InvisibleShaderName = "MixedRealityToolkit/InvisibleShader";
```

Comment:
Should this shader be configurable in the scriptable conifg

Reply:
No